src/renderer/components/Select/index.js

```diff
@@ -46,7 +46,7 @@ type Props = {
   rowHeight: number,
   error: ?Error, // NB at least a different rendering for now
   stylesMap: CreateStylesReturnType => CreateStylesReturnType,
-  extraRenderers?: { [string]: (props: *) => React$ElementType }, // Allows overriding react-select components. See: https://react-select.com/components
+  extraRenderers?: { disableTooltipText?: string, [string]: (props: *) => React$ElementType }, // Allows overriding react-select components. See: https://react-select.com/components
```

Comment:
IMHO `disableTooltipText`:
- should not be a property of `extraRenderers`, because `extraRenderers` has been made specifically to have keys corresponding to [react-select component names](https://react-select.com/components) - plus the object is spread and passed to react-select through the `components` prop and the additional keys will leak.
- have a different name since my understanding of `disableTooltipText` would be "disables the tooltip text" but I think that it actually "shows the tooltip text when the item is disabled"

So maybe we could pass it as a regular prop?

Reply:
@jgiamporcaro-ledger If you're ok with the change I can make a follow-up PR since this one has been merged already

Reply:
Yeah, I have no problem with the change.
In fact, the disableTooltipText is the string to pass to the tooltip component.

Reply:
> In fact, the disableTooltipText is the string to pass to the tooltip component.

Woops my bad then, name is fine!